### ISSUE TITLE
Screen Reader is not announcing all items of the context menu list nor does it read items in collapsed dropdown

### ISSUE THREAD

**Issue by smartguest** [microsoft/vscode#141316]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: N/A
- OS Version: Windows 10

## Context: 

Issues were originally reported here:
https://github.com/microsoft/vscode-mssql/issues/17200
https://github.com/microsoft/vscode-mssql/issues/17204

## Steps to Reproduce:

1. Run the screen reader NVDA or Jaws
2. Hover over an item in explorer or similar dropdowns in vscode (2nd issue)
3. Right click on the item and hover to the left of the first item, as shown here:
![Context Menu Screenshot](https://user-images.githubusercontent.com/18018452/150847886-12934042-a7e2-4179-b079-77ba25b53673.png) (1st issue)

## Actual results:

1st issue: Screen reader only announces the first item in the context menu and not the rest of the items.

2nd issue: Screen reader only announces the name of the item, and no indication that it contains items inside.

## Expected results:

1st issue: Screen reader should announce all items in context menu with positions such as 1 of 5, 2 of 5, etc when hovered over.

2nd issue: Screen reader should announce if an item in a dropdown has additional items inside it when hovered over.

**Comment by isidorn** [comment]
Screen readers are not announcing things based on hover but on focus. So can you please clarify the issue from above, and you can please file two issues if they are unrelated please?

Screen reader users do not use the mouse, so this sounds very invalid to me....

**Comment by smartguest** [comment]
@isidorn they are two similar issues with screen reader, also it's not necessary to click on an item on explorer (pressing shift + f10 will do the same), navigate it via arrow keys to the first item, the screen reader will then ONLY announce the first item and not the other ones below after a pause. (For more details see here: https://github.com/microsoft/vscode-mssql/issues/17200)

The second issue involves navigating to a folder item in explorer, and the screen reader not announcing anything about additional items inside, or that you can press shift-f10 to access the items inside (Again for more details see here: https://github.com/microsoft/vscode-mssql/issues/17204).

I think these issues are similar enough that they should be combined, unless they are separate, in which case please let me know.

These are issues I found with the vscode-mssql extension (which uses a similar folder and file browser) that also reproduce on the file explorer and other similar menus.

**Comment by isidorn** [comment]
@smartguest ok so the first feature request is to announce all menus items on menu focus. This is a fair request but something which I think will bring too much noise to the user experience. So no plans to add this until more users request it. Based on this renaming this issue.

Second item sounds like screen reader not announcing if an item is expanded or not. We set the aria-expanded state properly on the explorer item and aria-posinset on the children.
If these things are not being read I suggest you file an issue against your screen reader. Thanks

**Comment by smartguest** [comment]
@isidorn These were not my issues originally, they were brought up by @Yashi1201 during the review of our product. Please let him know because I'm merely relaying ones that also affect vscode.
